### PR TITLE
Added webcomics.json

### PR DESCRIPTION
**Pull request by Mav7**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer
    - [x] Cheat Sheets: **`New {Ruby Webcomics} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

I added a new cheat sheet for the ruby-webcomics library.

###### Which issues (if any) does this fix?
No issues to fix.

###### People to notify (@mention interested parties)

---

Instant Answer Page: https://duck.co/ia/view/ruby_webcomics_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mav7

### CODE REVIEW

**Comment by daxtheduck** [comment]
@Mav7 Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/webcomics.json [hunk 1]
```diff
@@ -0,0 +1,119 @@
+{
+    "id": "webcomics_cheat_sheet",
+    "name": "Webcomics",
+    "description": "displayed as subtitle of the AnswerBar; optional",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/webcomics.json]: Lets change the descriprition to some thing explaining contents of the cheat sheet

File: share/goodie/cheat_sheets/json/webcomics.json [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+    "metadata": {
+        "sourceName": "displayed at the bottom of the AnswerBar; optional",
+        "sourceUrl" : "https://url.to.the.source"
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/webcomics.json]: Change the source url and name

File: share/goodie/cheat_sheets/json/webcomics.json [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    },
+
+    "aliases": [
+        "first alias", "second alias", "etc"
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/webcomics.json]: We can have some aliases here if any, else remove this

File: share/goodie/cheat_sheets/json/webcomics.json [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+        "Comics Canvas",
+        "Comics Directory Helper",
+        "Comics Engine"
+    ],
+
+    "sections": {
+        "Comics Directory": [
+            {
+                "key": "return_an_array_of_comics_entries(directory)",
+                "val": "Returns an array of entries from a directory without previous and current directories."
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/webcomics.json]: Remove periods at end of all val fields

**Comment by Mav7** [comment]
@jophab I think I got all the mistakes out.

**Comment by jophab** [comment]
@Mav7  There should be only one file in the PRs for cheat sheets, the json file of this cheat sheet.
So remove others

**Comment by Mav7** [comment]
@jophab I don't think I know what you mean. webcomics.json should be the only file.

**Comment by moollaza** [comment]
> @jophab I don't think I know what you mean. webcomics.json should be the only file.

@Mav7 it looks like you may have made a mistake in the way you resolved your merge conflict.

I would suggest you make a new PR from a fresh branch that just adds your JSON file with a single commit. Your PR is showing 61 modified files instead of just the 1 so I'm hesitant to move forward here. Alternatively if you're capable of resetting your branch to be in line with master and adding a new commit here, you can do so to avoid opening a new PR

**Comment by Mav7** [comment]
@moollaza Ok I'll try the new PR.

**Pull request closed without merging**